dist/organizations-demo.json

```diff
@@ -0,0 +1,920 @@
+[
+  {
+    "id": "534fff4ca3a7292c64a77c95",
+    "name": "ADEME",
+    "slug": "ademe",
+    "type": ""
+  },
+  {
+    "id": "534fff4ea3a7292c64a77cad",
+    "name": "Agence de l'eau Rh\u00f4ne-M\u00e9diterran\u00e9e et Corse",
```

Comment:
Encoding issue. It should work, but it'd be better to have accents.

Reply:
It will work. I'm not sure it's an issue per se.

```
>>> json.dumps({"a": "éhoè"})
'{"a": "\\u00e9ho\\u00e8"}'
```

Reply:
Less human-friendly. Copy-pasting will be a pain for instance.

```
>>> json.dumps({"a": "éhoè"}, ensure_ascii=False)
'{"a": "éhoè"}'
```

That param evokes all kinds of headaches, but if we ever have weird chars that break under `ensure_ascii=False` it's likely we'll break at some other stage anyway...?

Reply:
You can copy paste it from the JS console :p But okay, we have notifications on failure anyway.

```
>>> json.dumps({"a": "éhoè✌️"}, ensure_ascii=False)
'{"a": "éhoè✌️"}'
```

Reply:
No JS console when looking at the github "api" page manually :shrug: 

Granted it won't be the main use for this file, but if we make it human-friendly (and it's a good idea since it's a reference), might as well make it all the way (provided it's mostly free to do so).